src/govuk/components/tag/_tag.scss

```diff
@@ -20,6 +20,8 @@
     text-decoration: none;
     text-transform: uppercase;
 
+    white-space: nowrap;
```

Comment:
This change seems unrelated to the commit description – if it's required, can we make this change in another commit that explains why it's being added, or possibly make it part of a separate PR?